### PR TITLE
Clone created

### PR DESCRIPTION
**Pull request by Vito-Lu**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+        this.name = name;
+        this.colour = colour;
+        this.type = type;
+        this.maxSpeed = maxSpeed;
+        this.acceleration = acceleration;
+        this.engine = engine;
+    }
+
+    @Override
+
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Car.java:22]: there shouldn't be any empty lines between annotation and method

File: src/main/java/core/basesyntax/Engine.java [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+                  String maxPower, String averagePower) {
+        this.name = name;
+        this.model = model;
+        this.type = type;
+        this.maxPower = maxPower;
+        this.averagePower = averagePower;
+    }
+
+    @Override
+
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Engine.java:20]: same

File: src/main/java/core/basesyntax/Car.java (hunk 1 above)
Comment (Vito-Lu) [reply src/main/java/core/basesyntax/Car.java]: done

File: src/main/java/core/basesyntax/Engine.java (hunk 2 above)
Comment (Vito-Lu) [reply src/main/java/core/basesyntax/Engine.java]: done

**Pull request closed without merging**